### DISCUSSION TITLE
LibreChatUI wont load after pulling latest docker + git for v0.7.4-rc1

### DISCUSSION THREAD

**Discussion by mtnsh** [danny-avila/LibreChat#3584] [Troubleshooting] [answered]
### What happened?

update both docker and git - UI wont load

### Steps to Reproduce

1 - docker compose down -> pull
2 - git pull
3 - docker compose up
4 - go to -> localhost:3080...

### What browsers are you seeing the problem on?

Microsoft Edge

### Relevant log output

```shell
couldn't find log anomalies for now
```

### Screenshots

keep loading, no response

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
> update both docker and git - UI wont load

Unfortunately I would need more info to properly help you troubleshoot this. I cannot reproduce this on a fresh clone down.

Can you try using the default `.env.example` file as your `.env` file temporarily to help troubleshoot, to see if the UI will load then?

You can also check your logs in the `./logs` directory from project root.

**Reply by danny-avila** [reply]
If you are also running mongodb locally (on your instance), you should check its container logs as well.

**Reply by mtnsh** [reply, marked as answer]
I've deleted and re-cloned.
run configuration again and it worked.
Thanks
